wolvic/wolvic_permission_manager.cc

```diff
@@ -269,10 +282,27 @@ void WolvicPermissionManager::OnPermissionResult(
                            return request.get() == in_progress_request;
                          });
   CHECK(it != in_progress_requests_.end());
-  std::move((*it)->callback).Run(result);
+  for (auto& callback : (*it)->callbacks) {
+    std::move(callback).Run(result);
+  }
   in_progress_requests_.erase(it);
 }
 
+InProgressRequest* WolvicPermissionManager::FindInProgressRequest(
+    const content::PermissionRequestDescription& description) {
+  auto it = std::find_if(in_progress_requests_.begin(),
+                         in_progress_requests_.end(), [&](const auto& request) {
+                           return request->description.requesting_origin ==
+                                      description.requesting_origin &&
+                                  request->description.permissions ==
+                                      description.permissions;
+                         });
+  if (it == in_progress_requests_.end()) {
+    return nullptr;
+  }
+  return it->get();
```

Comment:
maybe using ternary operator to do both returns in a single line?

Reply:
Fixed.